docs/UserGuide.md

```diff
@@ -188,7 +190,7 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [f/MONEY] [t/TAG]…​ [tk/TASK]` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague tk/Buy birthday cake`
```

Comment:
@yaololo as per my [comment](https://github.com/AY2122S2-TIC4002-F18-4/tp2/issues/52#issuecomment-1087699642), need to remove `Money` from `AddCommand`